packages/admin/admin-theme/src/componentsTheme/MuiFormLabel.ts

```diff
@@ -11,6 +11,18 @@ export const getMuiFormLabel: GetMuiComponentTheme<"MuiFormLabel"> = (component,
             lineHeight: "20px",
             fontWeight: typography.fontWeightBold,
             marginBottom: 4,
+
+            "&.Mui-disabled": {
+                color: palette.text.primary,
+            },
+
+            "&.Mui-focused": {
+                color: palette.text.primary,
+            },
+
+            "> .MuiFormLabel-asterisk": {
+                marginLeft: "4px",
+            },
```

Comment:
Please target `asterisk` directly, instead of selecting it as a child inside `root`

```suggestion
        },
        asterisk: {
            marginLeft: "4px",
        },
```